src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/AppointmentProcessor.kt

```diff
@@ -24,13 +24,13 @@ class AppointmentProcessor(
         appointmentTime = NdmisDateTime(it.appointmentTime),
         durationInMinutes = it.durationInMinutes,
         bookedAt = NdmisDateTime(it.createdAt),
-        didSessionHappen = it.didSessionHappen,
         attended = it.attended,
+        didSessionHappen = it.didSessionHappen,
```

Comment:
Just moved where this is in the list to be consistent with the AppointmentData file.